Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -650,6 +652,12 @@ CGImageRef _CGImageCreateCopyWithPixelFormat(CGImageRef image, WICPixelFormatGUI
     return imageRef;
 }
 
+CGImageRef _CGImageGetImageFromDataProvider(CGDataProviderRef provider) {
```

Comment:
Since you're introducing a totally new function, can you name it `_CGImageCreateFromDataProvider`? While the other `Get` is wrong, this one doesn't have to be!